src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -48,9 +49,6 @@
     @FXML
     private StackPane commandBoxPlaceholder;
 
-    @FXML
-    private MenuItem helpMenuItem;
```

Comment:
Since we are removing the menu, I think you can remove the corresponding CSS too!